frontend/src/components/recommitment/RecommitmentFormBase.tsx

```diff
@@ -320,8 +328,15 @@ export const RecommitmentFormBase = ({
     const decision =
       decisionMap[recommitmentAnswer as keyof typeof decisionMap] || {};
     console.log(decision);
-    await updateRecommitment(personnel.id, { ...decision, supervisorInformation });
-    onClose();
+    const res = await updateRecommitment(personnel.id, {
+      ...decision,
+      supervisorInformation,
+    });
+    if (res.error) {
```

Comment:
Don't close if we get an error, show the banner so user knows the form was not submitted